t/plugin/chaitin-waf.t

```diff
@@ -270,3 +271,205 @@ X-APISIX-CHAITIN-WAF-STATUS: 200
 X-APISIX-CHAITIN-WAF-ACTION: pass
 --- response_headers_like
 X-APISIX-CHAITIN-WAF-TIME:
+
+
+=== TEST 8: plugin mode off
```

Comment:
@membphis Hello, I have a quick question, I am really having a hard time setting the routes in my test cases I am doing something obviously wrong here ?

Reply:
To be more descriptive the mode never gets set to off and it seems I cannot set the uri config for route 1